src/Microsoft.Devices.HardwareDevCenterManager/Abstractions/IDevCenterHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*++
+    Copyright (c) Microsoft Corporation. All rights reserved.
+
+    Licensed under the MIT license.  See LICENSE file in the project root for full license information.  
+--*/
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.Devices.HardwareDevCenterManager.DevCenterApi
+{
+    public interface IDevCenterHandler
+    {
+        Task<DevCenterResponse<bool>> CommitSubmission(string productId, string submissionId);
+        Task<DevCenterResponse<bool>> CreateMetaData(string productId, string submissionId);
+        void Dispose();
```

Comment:
I don't think the `IDevCenterHandler` interface needs this `Dispose` method. the `DevCenterHandler `concrete implementation implements `IDisposable` but that should not bleed to this interface.

Reply:
I've removed Dispose() from this interface